gpg-check.rst

```diff
@@ -0,0 +1,54 @@
+Verifying GPG signature of Electrum using Linux command line
+============================================================
+
+This can be used to verify either the the Linux Python or AppImage version of Electrum
```

Comment:
This guide can be used to verify any download not just the Linux ones
```suggestion
This can be used to verify the authenticity of Electrum binaries/sources.
```